src/main/java/challenges/BinarySearch.java

```diff
@@ -0,0 +1,38 @@
+package challenges;
+
+public class BinarySearch {
+    public static void main (String[] arg){
+        //tests
+        System.out.println(binarySearch(new int[] {2, 4, 6, 8, 10}, 7));

```

Comment:
You'll need to start writing your tests in a separate file using JUNit. Let me know if you have questions